core/files/css/global-css.php

```diff
@@ -93,21 +92,7 @@ protected function get_inline_dependency() {
 	 * @return bool True if the CSS requires an update, False otherwise.
 	 */
 	protected function is_update_required() {
-		$file_last_updated = $this->get_meta( 'time' );
-
-		$schemes_last_update = get_option( Scheme_Base::LAST_UPDATED_META );
-
-		if ( $file_last_updated < $schemes_last_update ) {
-			return true;
-		}
-
-		$elementor_settings_last_updated = get_option( Settings::UPDATE_TIME_FIELD );
-
-		if ( $file_last_updated < $elementor_settings_last_updated ) {
-			return true;
-		}
-
-		return false;
+		return $this->get_meta( 'time' ) < get_option( Settings::UPDATE_TIME_FIELD );
```

Comment:
On a refresh install both values are empty, and it will return false even the update is required.

Reply:
It's exactly like the previous check. Only the schemes file check was removed

Reply:
Currently, on a fresh install it will return false.

Reply:
Can you provide me with a real case (with steps to reproduce) when you encounter a problem as a result of this behavior?

Reply:
OK, I checked and a fresh install is covered by another logic.